### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by synicalsyntax**
#49

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"use strict"; // catch errors easier
+"use strict"; // catch errors easierf
```
Comment (synicalsyntax) [inline src/index.js]: Fix that

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development members, this pull request was labeled with the **area: documentation (developer)** label, so you may want to check it out!

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-"use strict"; // catch errors easier
```
Comment (synicalsyntax) [inline src/index.js]: hi

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-# test
```
Comment (synicalsyntax) [inline README.md]: DSFSDF

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "zulipbot",
+  "name": "zuldipbot",
```
Comment (zulipbot-test) [inline package.json]: Bad idea

**Comment by synicalsyntax** [comment]
#49

File: src/index.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+"use strict"; // catch errors easierf
```
Comment (synicalsyntax) [inline src/index.js]: wefwef

File: src/index.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
...
 
 // requirements
 const express = require("express");
```
Comment (synicalsyntax) [inline src/index.js]: sdfsdfsfsfsf

**Comment by synicalsyntax** [comment]
#49

File: package.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "zulipbot",
```
Comment (synicalsyntax) [inline package.json]: Hello

File: package.json [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (synicaltests) [inline package.json]: nesoberi

**Pull request closed without merging**